tests/topotests/ospfv3_basic_functionality/test_ospfv3_rte_calc.py

```diff
@@ -273,7 +273,7 @@ def red_connected(dut, config=True):
         ospf_red = {
             dut: {
                 "ospf6": {
-                    "redistribute": [{"redist_type": "connected", "del_action": True}]
+                    "redistribute": [{"redist_type": "connected", "delete": True}]
```

Comment:
Same comment as above. This is calling `create_router_ospf` which did not do anything with the previous `del_action` key, but now will with the `delete` key. Yet the tests passed previously.